### DISCUSSION TITLE
dependency on an unsupported fork

### DISCUSSION THREAD

**Discussion by DogFox** [element-plus/element-plus#11321] [Q&A]
"@popperjs/core": "npm:@sxzz/popperjs-es@^2.11.7"

Its a fork of floating ui, which included in dependency.
Looks like not best solution

**Comment by puzirev** [comment]
I'm absolutely shure that usage of original floating-ui is much better solution.
If we realy need to change packaging method of floating-ui, we can add it as submodule and use it's sources or just repack the packet in our build process. In my opinion fork is the worst option.

**Comment by rickyPanzer** [comment]
Seems like this has consequences now, currently using el-popover from element-plus 2.4.3. When using the trigger option, getting this error when upgrading from vue 2 to vue 3. Seems like this popover fork is around 2 years old now. Would be great to see the transition to floating-ui!

index.ts:11 [Vue warn]: (deprecation ATTR_FALSE_VALUE) Attribute "data-popper-reference-hidden" with v-bind value `false` will render data-popper-reference-hidden="false" instead of removing it in Vue 3. To remove the attribute, use `null` or `undefined` instead. If the usage is intended, you can disable the compat behavior and suppress this warning with:
```
  configureCompat({ ATTR_FALSE_VALUE: false })

  Details: https://v3-migration.vuejs.org/breaking-changes/attribute-coercion.html 
  at <ElPopperContent key=0 id="el-id-9720-1" ref_key="contentRef"  ... > 
  at <BaseTransition onAfterLeave=fn<onTransitionLeave> onBeforeEnter=fn<onBeforeEnter> onAfterEnter=fn<onAfterShow>  ... > 
  at <Transition name="el-fade-in-linear" onAfterLeave=fn<onTransitionLeave> onBeforeEnter=fn<onBeforeEnter>  ... > 
  at <ElTooltipContent ref_key="contentRef" ref=Ref< Proxy(Object) {contentRef: RefImpl, __v_skip: true} > aria-label=undefined  ... > 
  at <ElPopper ref_key="popperRef" ref=Ref< Proxy(Object) {triggerRef: RefImpl, popperInstanceRef: RefImpl, contentRef: RefImpl, referenceRef: RefImpl, role: ComputedRefImpl, …} > role="tooltip"  ... > 
  at <ElTooltip ref_key="tooltipRef" ref=Ref< Proxy(Object) {popperRef: RefImpl, contentRef: RefImpl, isFocusInsideContent: ƒ, updatePopper: ƒ, onOpen: ƒ, …} > modelValue=false  ... > 
  at <ElPopover key=0 placement="bottom" width=100  ... > 
  at <CustomHeader checkboxOptions=null currentSort= {prop: 'name', order: 'ascending'} filterFunction=fn<bound filterByNumber>  ... > 
  at <ElTableHeader ref="tableHeaderRef" border=false default-sort=undefined  ... > 
  at <ElTable data= [] onSortChange=fn<bound changeTableSort> summary-method=fn<bound sumTableColumns>  ... > 
  at <Home onVnodeUnmounted=fn<onVnodeUnmounted> ref=Ref< Proxy(Object) {type_filter: ƒ, reservesTotal_filter: ƒ, observers_filter: ƒ, name_filter: ƒ, luxOwnership_filter: ƒ, …} > > 
  at <KeepAlive key=0 > 
  at <RouterView> 
  at <ElMain> 
  at <ElContainer> 
  at <ElContainer> 
  at <App>
```

the follow up error seems to be more damning, I will say:
```
The above deprecation's compat behavior is disabled and will likely lead to runtime errors.
```
